R/contingencytables.R

```diff
@@ -65,6 +65,12 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
              type = c('negativeValues', 'infinity', 'missingValues'),
              all.target = c(options$counts),
              exitAnalysisIfErrors = TRUE)
+
+  .hasErrors(dataset,
+             type = "factorLevels",
+             factorLevels.target = c(options[["rows"]], options[["columns"]]),
+             factorLevels.amount  = "< 2",
+             exitAnalysisIfErrors = TRUE)
```

Comment:
I wonder if we shouldn't do this on a per-analysis basis.
I feel like this is a dilemma we keep returning to: keep the code simpler but deliver fewer results (including those that could be computed), or deliver as many valid results as possible while excluding the invalid ones

Reply:
But, honestly, I don't think it's a good use of your time considering the state of the code in this analysis